### ISSUE TITLE
QWebview customContextMenuRequested signal

### ISSUE THREAD

**Issue by Alexpux** [qtwebkit/qtwebkit#1075]
Using mingw version of QWebview 5.212.0alpha4 I cant show context menu on page using  **customContextMenuRequested** signal.

```
QWebView *webview = new QWebView(this);
webview->setContextMenuPolicy(Qt::CustomContextMenu);
connect(webview, &QWebView::customContextMenuRequested, this, &Form::showWebViewContextMenu);
```

When right button click on webview nothing happen and **showWebViewContextMenu** procedure not executed.

**Comment by annulen** [comment]
cc @Emdek

**Comment by Emdek** [comment]
@Alexpux, could you please provide some more details?
I can try to reproduce it later using MXE and Windows 10.

**Comment by Alexpux** [comment]
@Emdek I want to create context menu with some actions for current page. As I wrote previously signal customContextMenuRequested doesn't fired when I clicked right mouse button.

**Comment by Emdek** [comment]
@Alexpux, I should be more precise, I meant stuff like platform where you are running resulting binary, is it exactly alpha4 tag (I'm using post alpha4 build) etc.

**Comment by Alexpux** [comment]
@Emdek Yes I'm running alpha4 tag on Win10, builder with Mingw-w64 and Qt 5.15.2
